src/Mgate/StatBundle/Controller/IndicateursController.php

```diff
@@ -116,24 +117,23 @@ private function getRetardParMandat()
         $categories = [];
         foreach ($nombreJoursParMandat as $idMandat => $datas) {
             if ($datas > 0) {
-                $categories[] = $idMandat;
+                $categories[] = $etudeManager->mandatToString($idMandat);
```

Comment:
Not ok in N7 Consulting context: our mandat field is directly the year of beginning (2017 for instance). We've been created in 1977. With such call, it will display 3994.

Reply:
Misunderstood your mandat field: I thought it was based on creation year like us but you preferred to see it as a year. Then it would have fit for both if displayed as X/Y